app/javascript/src/components/Common/Dropdown.jsx

```diff
@@ -1,16 +1,16 @@
 import React from "react";
 
 import { MenuVertical } from "neetoicons";
-import { Dropdown } from "neetoui";
+import { Dropdown as NeetoUIDropdown } from "neetoui";
 
-const NoteDropDown = ({ setShowDeleteAlert }) => {
-  const { Menu, MenuItem } = Dropdown;
+const Dropdown = ({ setShowDeleteAlert }) => {
+  const { Menu, MenuItem } = NeetoUIDropdown;
```

Comment:
Move this outside of the component declaration so that this statement doesn't have to be re-run when the component re-renders.